docs/csi_ug_about_this_guide.md

```diff
@@ -0,0 +1,17 @@
+# About this guide
+
+This guide describes how to install, configure, and use the IBM® block storage CSI driver.
+
+-   **[Who should use this guide](csi_ug_audience.md)**  
+This guide is intended for system administrators who are familiar with container-based application delivery, orchestration methods, and with the specific IBM storage system that is in use.
+-   **[Conventions used in this guide](csi_bookconventions.md)**  
+These notices are used in this guide to highlight key information.
+-   **[Related information and publications](csi_relatedpubs.md)**  
+For more information storage systems and orchestration platforms that are supported by IBM block storage CSI driver, see the following resources.
+-   **[Getting information, help, and service](csi_gettinginfohelpandservice.md)**  
+
+-   **[IBM Publications Center](csi_howtoorderpubs.md)**  
+The IBM Publications Center is a worldwide central repository for IBM product publications and marketing material.
+-   **[Sending comments](../../../../StorageCommonFiles/common_topics/sending_comments.md)**  
```

Comment:
Broken link

Reply:
removed this file from the docs/ folder. it is not needed there. it's in the book_files folder as a dita file